binary_search/Python/binary_search.py

```diff
@@ -0,0 +1,50 @@
+
+def binarySearch (arr, l, r, key): 
```

Comment:
make use of snake case. `binarySearch` is camel case.
Also, remove whitespace before opening parenthesis.